### ISSUE TITLE
Nuget.exe update reinstall

### ISSUE THREAD

**Issue by mj1856** [NuGet/Home#341]
The powershell cmdlet `Update-Package` has the `-reinstall` flag, [as described here](https://docs.nuget.org/consume/reinstalling-packages).

It would be nice if this functionality was also available from `nuget.exe` as a parameter to the `update` command.

**Comment by davidfowl** [comment]
`nuget.exe update` is an abomination though, adding stuff to it just seems like a bad idea. Are you updating project files from the command line outside of visual studio often?

**Comment by mj1856** [comment]
No, not often.  We came across a need to re-install a package during the build process.  The package unfortunately contains a few .exe files as content, which we don't want in source control.  Reinstall from VS package manager console fixes the problem, except when starting clean.

We're looking at other approaches as well, but one of the first things I tried was just to invoke the update as a prebuild step, which is where I found that `-reinstall` wasn't available on nuget.exe.
